proxy/InkAPI.cc

```diff
@@ -8941,3 +8942,110 @@ TSVConnReenable(TSVConn vconn)
     }
   }
 }
+
+// APIs for managing and using UUIDs.
+TSUuid
+TSUuidCreate(void)
+{
+  ATSUuid *uuid = new ATSUuid();
+  return (TSUuid)uuid;
+}
+
+TSReturnCode
+TSUuidDestroy(TSUuid uuid)
+{
+  sdk_assert(sdk_sanity_check_null_ptr((void *)uuid) == TS_SUCCESS);
+  ATSUuid *u = (ATSUuid *)uuid;
+
+  if (u->alive()) {
```

Comment:
It used to do more, but not it just does a poor amount of sanity check, and it certainly can not be a guarantee. It could not be a false negative though. It'd probably make more sense if the data was on a freelist, maybe.